<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cars</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Label_Power">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Power (horsepower) :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Seats">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Seats :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Category">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Category :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Consumption">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Consumption(l/100km) :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_GearBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gear Box :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Prices">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Prices (â‚¬) :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Power">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Seats">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Category">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Consumption">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_GearBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Prices">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="RunButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Validate</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Car">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
